eng/scripts/Test-Packages.ps1

```diff
@@ -62,7 +62,7 @@ foreach ($package in $packagesToTest) {
     Invoke-LoggedCommand "cargo test --doc --no-fail-fast" -GroupOutput
     Write-Host "`n`n"
 
-    Invoke-LoggedCommand "cargo test --all-targets --no-fail-fast" -GroupOutput
```

Comment:
@heaths as discussed offline, I've put this in my PR for now, since I need it to verify our tests are running anyway.